editor.planx.uk/src/@planx/components/List/Public/index.tsx

```diff
@@ -25,7 +25,7 @@ import { ListProvider, useListContext } from "./Context";
 
 export type Props = PublicProps<List>;
 
-const ListCard = styled(Box)(({ theme }) => ({
+export const ListCard = styled(Box)(({ theme }) => ({
```

Comment:
nit: Maybe this should be in a shared location (shared/Schema/styles.tsx?) instead of importing directly from the List component. Without this, we could make a change here without really realising it has impacts elsewhere.

Reply:
Thanks for catching this - no longer exported at all! This was a remnant from the first non-tab prototype where we just showed a `ListCard` for each feature on the map, but now we've got the tabs